settings.gradle

```diff
@@ -1,12 +1,31 @@
+buildscript {
+    ext.apollo_version = '2.5.3'
```

Comment:
Seems like apollo_version is defined twice, which kinda defeats the purpose of using a variable